sdks/unity/Tests/Runtime/PlayMode/AgonesSdkComplianceTests.cs

```diff
@@ -0,0 +1,33 @@
+using System.Collections;
+using System.Threading.Tasks;
+using Agones;
+using UnityEngine;
+using UnityEngine.Assertions;
+using UnityEngine.TestTools;
+
+namespace Tests.Runtime.Playmode
+{
+    public class AgonesSdkComplianceTests
+    {
+        [UnityTest]
+        public IEnumerator AgonesSdk_Ready_AlwaysReturnsTrue()
+        {
+            var gameObject = new GameObject();
+            var sut = gameObject.AddComponent<AgonesSdk>();
+            var task = sut.Ready();
+            yield return AwaitTask(task);
+            Assert.IsTrue(task.Result);
```

Comment:
This is not a bad test (although we may get double coverage if we ever do a conformance test - which is not well documented)  - but wondering if this passes, as there is no mock for the http endpoint for the `/ready` request?